feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -183,7 +183,7 @@ func TestBasic(t *testing.T) {
 				EqualToDefault(sysCounts.AuthFails().State(), uint32(0)),
 				EqualToDefault(sysCounts.AuthTypeFails().State(), uint32(0)),
 				EqualToDefault(sysCounts.CorruptedLsps().State(), uint32(0)),
-				check.Present[uint32](sysCounts.DatabaseOverloads().State()),
+				EqualToDefault(sysCounts.DatabaseOverloads().State(), uint32(0)),
```

Comment:
database-overloads leaf does’nt have any default value in the model & this leaf can be set by different ways [set-overload bit, after reload, config  etc] based on Implementation.
So it’s not correct to assume value is either 0 or nil , and it can be ≥ 0 as well.

Reply:
Updated code to perform `check.Present` only when `MissingValueForDefaults` is marked false.